tests/hardhat/Fork/RewardsForkTest.ts

```diff
@@ -92,38 +80,41 @@ async function grantPermissions() {
   await tx.wait();
 }
 
-if (FORK_TESTNET || FORK_MAINNET) {
+if (FORK) {
```

Comment:
idea: If these tests only run when running the fork tests you could create an entrypoint file that isn't run with the normal tests and the point to that i the hardhat test command when running fork tests. The fork tests would need to go in a different directory like `fork-test` for example

so you would have `yarn hardhat test` for unit tests and `yarn hardhat test fork-test/index.ts` for fork tests

Reply:
The index.ts could just import each fork test in the order to run them
```
import './RewardsForkTest.ts'
...
```

Reply:
We have the script to run all fork tests at once
    `yarn test:fork`
          Or we can use
    `npx hardhat test tests/hardhat/Fork/*`

Wdyt, should we create index.ts?

Reply:
I think `npx hardhat test tests/hardhat/Fork/*` could also work. My motivation in suggesting these changes was to refactor the tests out of if statements and separate running fork and unit tests into two different commands that don't mix the tests

Reply:
I will prefer to go with the current changes for now. If we need to do these changes we can obviously do them in another ticket.